src/snapshot/cloneplot.js

```diff
@@ -115,9 +115,21 @@ module.exports = function clonePlot(graphObj, options) {
         for(i = 0; i < sceneIds.length; i++) {
             var sceneId = sceneIds[i];
 
-            extendFlat(newLayout[sceneId].xaxis, axesImageOverride);
-            extendFlat(newLayout[sceneId].yaxis, axesImageOverride);
-            extendFlat(newLayout[sceneId].zaxis, axesImageOverride);
+            if(!newLayout[sceneId].xaxis) {
+                newLayout[sceneId].xaxis = {};
+            }
+
+            if(!newLayout[sceneId].yaxis) {
+                newLayout[sceneId].yaxis = {};
+            }
+
+            if(!newLayout[sceneId].zaxis) {
+                newLayout[sceneId].zaxis = {};
+            }
+
+            extendFlat(newLayout[sceneId].xaxis || {}, axesImageOverride);
+            extendFlat(newLayout[sceneId].yaxis || {}, axesImageOverride);
+            extendFlat(newLayout[sceneId].zaxis || {}, axesImageOverride);
```

Comment:
`|| {}` is unnecessary here (and would have been confusingly useless by itself)

I'd probably also do `var scene = newLayout[sceneIds[i]]` instead of `var sceneId = sceneIds[i]` to be more concise, but that's more of a style issue.